WeCantSpell.Hunspell/WordList.QuerySuggest.cs

```diff
@@ -1,111 +1,141 @@
 ﻿using System;
+using System.Buffers;
 using System.Collections.Generic;
+using System.Globalization;
 using System.Linq;
-using System.Text;
 
 using WeCantSpell.Hunspell.Infrastructure;
 
-#if !NO_INLINE
-using System.Runtime.CompilerServices;
-#endif
-
 namespace WeCantSpell.Hunspell;
 
 public partial class WordList
 {
-    private sealed class QuerySuggest : Query
+    private struct QuerySuggest
     {
-        /// <summary>
-        /// Timelimit: max ~1/4 sec (process time on Linux) for a time consuming function.
-        /// </summary>
-        private const int TimeLimitMs = 1000 >> 2;
-
-        private const int MinTimer = 100;
-
+        private const int MaxPhoneTLen = 256;
+        private const int MaxPhoneTUtf8Len = MaxPhoneTLen * 4;
+        [Obsolete("I'm not sure what this is for.")]
         private const int MaxPlusTimer = 100;
```

Comment:
Maybe this should be removed? Research it in the origin codebase first.

Reply:
This is being moved to the timer class and also to better match timeout behavior